recipes/komb/meta.yaml

```diff
@@ -0,0 +1,41 @@
+package:
+  name: komb
+  version: "1.0"
+
+source:
+  url: https://gitlab.com/treangenlab/komb/-/archive/v1.0/komb-v1.0.tar.gz
```

Comment:
This link looks wired. Can you use the classical one and replace the version number with a templated version?